tests/test_accelerator.py

```diff
@@ -73,6 +87,23 @@ def forward(self, x):
         return self.linear2(self.batchnorm(self.linear1(x)))
 
 
+def create_dataloaders_for_test(
+    a=2, b=3, batch_size=3, n_train_batches: int = 12, n_valid_batches: int = 2, num_workers=0
```

Comment:
What is with the `a` and `b` parameters? Can they be removed?

Reply:
I think that's leftover from a different approach, removed